Framework/CurveFitting/src/Algorithms/Fit.cpp

```diff
@@ -48,6 +48,9 @@ void Fit::initConcrete() {
       ->setDocumentation("Whether the fit was successful");
   declareProperty("OutputChi2overDoF", 0.0, "Returns the goodness of the fit",
                   Kernel::Direction::Output);
+  declareProperty("OutputNIterations", 0, Kernel::Direction::Output);
```

Comment:
The tuples still allow index access (or we would have broken scripts when doing that).

I think we should move this to the end of output list to keep things compatible.